core/dbt/include/global_project/macros/materializations/models/materialized_view/refresh_materialized_view.sql

```diff
@@ -0,0 +1,10 @@
+{% macro refresh_materialized_view(relation, config) %}
```

Comment:
I doubt we'll need `config` here. We'll need it in determining *if* we need to refresh, but this should just be the logic to refresh, with the assumption that we've verified that we have to refresh prior to calling this macro.